elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchSchemaFactory.java

```diff
@@ -88,7 +92,9 @@ public ElasticsearchSchemaFactory() {
       }
       final String pathPrefix = (String) map.get("pathPrefix");
       // create client
-      final RestClient client = connect(hosts, pathPrefix);
+      String jdbcUser = (String) map.get("jdbcUser");
+      String jdbcPassword = (String) map.get("jdbcPassword");
+      final RestClient client = connect(hosts, pathPrefix, jdbcUser, jdbcPassword);
```

Comment:
Given that this refers to the username and password for Elasticsearch and is not directly related to JDBC, this should be changed to `username` and `password`.